### PR TITLE
BAM-5249: Brand Tag Updates

### PR DESCRIPTION
**Pull request by nickcroce**
Updates from myself and the business

### CODE REVIEW

File: docs/tagmerchant.rst [hunk 1]
```diff
@@ -1,152 +1,163 @@
...
+Checkouts
+^^^^^^^^^
 
-Adding Information About Your Checkout Page
--------------------------------------------
+Checkout events allow Narrativ to accurately capture revenue and conversions in real time.
+The data gathered by checkout events also powers our Cart Explorer, a tool that publishers can use
+to give better product recommendations on their articles.
 
+Checkout events should be fired on your site’s **Order Confirmation Page**, on the page that loads
```
Comment (Asadius) [inline docs/tagmerchant.rst:24]: , or the page?

File: docs/tagmerchant.rst [hunk 2]
```diff
@@ -1,152 +1,163 @@
...
+Checkout Events
+^^^^^^^^^^^^^^^
 
-   * - products_purchased
-     - array
-     - Required. An array of Product objects representing the purchased items, as defined below.
+In order to successfully fire checkout events with the Narrativ Brand Tag, the following code snippet
+must be populated with information about the checkout, along with a few details about each of
+the products purchased. This snippet should be placed on your site’s **Order Confirmation Page**,
+or the page displayed to customers immediately after they’ve *successfully* purchased their items.
```
Comment (Asadius) [inline docs/tagmerchant.rst:67]: consistent with my previous comment

File: docs/tagmerchant.rst [hunk 3]
```diff
@@ -1,152 +1,163 @@
...
+The data gathered by checkout events also powers our Cart Explorer, a tool that publishers can use
+to give better product recommendations on their articles.
 
+Checkout events should be fired on your site’s **Order Confirmation Page**, or the page that loads
+immediately **after** they’ve *successfully* purchased their items.
 
-For checkout events, you have to fill in some information about the checkout before loading our Javascript. This is
-done by inserting the relevant data into a variable called ``window.BAMX_EVENT_DATA``. If a field is optional, you
-can leave it out if you choose not to include the data.
+**Please note:** No Personally Identifiable Information (PII) is captured by the Narrativ Brand Tag.
```
Comment (kaitlynkrull) [inline docs/tagmerchant.rst:27]: Do we need to mention this twice? It's on L15 too

File: docs/tagmerchant.rst [hunk 4]
```diff
@@ -1,152 +1,163 @@
...
 
-   * - page_type
-     - string
-     - Required. Must be "checkout" to tell the tag to fire a checkout event.
+* Make sure to replace `ACCOUNT NAME` with your Narrativ account name.
 
-   * - user_id
-     - string
-     - Optional. The ID the user has in your system, if available.
+* Need your Narrativ account name? Reach out to your growth manager or to `support@narrativ.com` for assistance.
```
Comment (kaitlynkrull) [inline docs/tagmerchant.rst:44]: can remove the second "to"

File: docs/tagmerchant.rst (hunk 3 above)
Comment (nickcroce) [reply docs/tagmerchant.rst]: Yep!

**Pull request merged**